contracts/wallet.sol

```diff
@@ -450,7 +450,7 @@ contract LoadLimit is ControllableOwnable, SelfCallableOwnable {
     event SetLoadLimit(address _sender, uint _amount);
     event SubmittedLoadLimitUpdate(uint _amount);
 
-    uint constant private _MINIMUM_LOAD_LIMIT = 1 finney;
+    uint private _minimumLoadLimit;
```

Comment:
Given that this is going to always be zero now, I'd prefer if we just took this variable out and make the contract cheaper to deploy.